### ISSUE TITLE
Http 426 - Upgrade Required 

### ISSUE THREAD

**Issue by AlanAds03** [trydis/FIFA-Ultimate-Team-Toolkit#402]
I am needing some requests to return Http 426 - Upgrade Required and even forcing a new Login I get a message from Temporary Ban 521.
Someone would know how to tell me what happens?

**Comment by InkedCurtis** [comment]
@AlanAds03 Are you running this on localhost?

**Comment by AlanAds03** [comment]
yes localhost, I'm afraid to see with pinEvents, I'm doing tests.
